board/piksiv3/rootfs-overlay/etc/init.d/S18cfg

```diff
@@ -8,6 +8,13 @@ case "$1" in
   mkdir -p /cfg
   SBP_SENDER_ID_HEX=`cat /factory/uuid | tail -c 4`
   printf "%u" "0x$SBP_SENDER_ID_HEX" > /cfg/sbp_sender_id
+  # Roll your own formatted device UUID
+  UUID_P0=`cat /factory/uuid | cut -c1-8`
+  UUID_P1=`cat /factory/uuid | cut -c9-12`
+  UUID_P2=`cat /factory/uuid | cut -c13-16`
+  UUID_P3=`cat /factory/uuid | cut -c17-20`
+  UUID_P4=`cat /factory/uuid | cut -c21-32`
+  printf "%s-%s-%s-%s-%s" "$UUID_P0" "$UUID_P1" "$UUID_P2" "$UUID_P3" "$UUID_P4" > /cfg/device_uuid
```

Comment:
This was brought up earlier when it was going in. I think there are dependencies on this now - e.g., `PKUUID` is used for TFTP bootloading. I'd be happy not to have to do this, but I think it might be stuck like this - @jacobmcnamee ?